### PR TITLE
Add rpi5 support to hello_snmp_manager

### PR DESCRIPTION
**Pull request by pojiro**
~~I don't have snmp device. I'm thinking about how to test.~~

I can find my snmp device. These are the results, works on both `rpi5` and `host`.

```elixir
iex(3)> Nerves.Runtime.mix_target
:rpi5

23:29:53.775 [debug] Sending SNMP Request for System Information

23:29:53.776 [debug] SNMP Response Successful

23:29:53.776 [debug] System Name: WAB04AB18849D92

23:29:53.776 [debug] System Description: ELECOM WAB-I1750-PS Ver2.0.4i
```

```elixir
iex(1)> Nerves.Runtime.mix_target
:host

08:32:57.345 [debug] Sending SNMP Request for System Information

08:32:57.356 [debug] SNMP Response Successful

08:32:57.356 [debug] System Name: WAB04AB18849D92

08:32:57.356 [debug] System Description: ELECOM WAB-I1750-PS Ver2.0.4i
```

### CODE REVIEW

File: hello_snmp_manager/lib/hello_snmp_manager.ex [hunk 1]
```diff
@@ -33,7 +33,7 @@ defmodule HelloSnmpManager do
   def handle_info(:tick, state) do
     Logger.debug("Sending SNMP Request for System Information")
 
-    :snmpm.sync_get('default_user', 'default_agent', [
+    :snmpm.sync_get2(~c"default_user", ~c"default_agent", [
```
Comment (pojiro) [inline hello_snmp_manager/lib/hello_snmp_manager.ex:36]: `snmpm:sync_get/3` was removed in OTP25.
ref. https://www.erlang.org/docs/23/general_info/scheduled_for_removal

File: hello_snmp_manager/config/config.exs [hunk 2]
```diff
@@ -20,12 +20,6 @@ config :nerves, :firmware, rootfs_overlay: "rootfs_overlay"
 
 config :nerves, source_date_epoch: "1619044926"
 
-# Use Ringlogger as the logger backend and remove :console.
-# See https://hexdocs.pm/ring_logger/readme.html for more information on
-# configuring ring_logger.
-
-config :logger, backends: [RingLogger]
-
```
Comment (pojiro) [inline hello_snmp_manager/config/config.exs:28]: moved this to `target.exs` for host log.

**Comment by pojiro** [comment]
@fhunleth I could confirm `hello_snmp_manager` on rpi5. Thanks.

**Comment by fhunleth** [comment]
Great! Thank you for testing and fixing the OTP25 change!

**Pull request merged**